t/ui/02-list-group.t

```diff
@@ -0,0 +1,67 @@
+# Copyright (C) 2014 SUSE Linux Products GmbH
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+BEGIN {
+    unshift @INC, 'lib';
+}
+
+use Mojo::Base -strict;
+use Test::More;
+use Test::Mojo;
+use OpenQA::Test::Case;
+
+OpenQA::Test::Case->new->init_data;
+
+use t::ui::PhantomTest;
+
+my $t = Test::Mojo->new('OpenQA');
+
+my $driver = t::ui::PhantomTest::call_phantom();
+if ($driver) {
+    plan tests => 8;
+}
+else {
+    plan skip_all => 'Install phantomjs to run these tests';
+    exit(0);
+}
+
+#
+# List with no parameters
+#
+is($driver->get_title(), "openQA", "on main page");
+my $baseurl = $driver->get_current_url();
+
+#
+# Test the legacy redirection
+#
+is(1, $driver->get($baseurl . "tests"), "/tests");
```

Comment:
Is there some reason to do this like that? If ok(..) is not suitable, at least please put expected value as second param. Else when the test fail you get i.e. expected undef, but got 1.